examples/rh-sso/client-registrations/webhook_handler.lua

```diff
@@ -0,0 +1,105 @@
+local redis_pool = require 'client-registrations/redis_pool'
+local lom = require 'lxp.lom'
```

Comment:
It has [rockspec](https://github.com/Phrogz/SLAXML/blob/master/slaxml-0.7-0.rockspec) so maybe it just has to be pushed.